ui/lib/apps/InstanceProfiling/translations/en.yaml

```diff
@@ -23,6 +23,7 @@ instance_profiling:
         finished: Finished
         failed: Failed
         partial_finished: Partial Finished
+        legacy_all_finished: Partial Or All Finished
```

Comment:
Actually this description might not be correct.

Consider a case that all sub tasks are failed. Previously this will fall into "Finished". But it is not partial finish at all. None is finished.

What we can only for sure is that, in such case, there is at least one failure.